apps/tandoor/docker-compose.yml

```diff
@@ -0,0 +1,60 @@
+version: "3.7"
+services:
+  tandoor:
+    container_name: tandoor
+    image: ghcr.io/tandoorrecipes/recipes:1.5.4
+    volumes:
+      - ${APP_DATA_DIR}/data/staticfiles:/opt/recipes/staticfiles
+      - ${APP_DATA_DIR}/data/mediafiles:/opt/recipes/mediafiles
```

Comment:
maybe we can remove "files" for app data dir, as it's redundant.
```
- ${APP_DATA_DIR}/data/staticfiles:/opt/recipes/static
- ${APP_DATA_DIR}/data/mediafiles:/opt/recipes/media
```
why not also use /runtipi/media/recipes @meienberger ?

Reply:
I don't know about this one...

Reply:
Now that I rethinked it, it's not exactly media so probably no..